worldedit-core/src/main/java/com/sk89q/worldedit/math/convolution/HeightMapFilter.java

```diff
@@ -80,50 +80,109 @@ public void setKernel(Kernel kernel) {
      * @return the modified height map
      */
     public int[] filter(int[] inData, int width, int height) {
+        return filter(inData, width, height, 0.5F);
```

Comment:
I'm still not sure why you added this override:
> I am not entirely sure why you added this method, the only use of it also passed 0.5F?

Reply:
Same method signature as filter(float[]...) and it doesn't hurt. maybe someone want's to use a different offset as 0.5?

Reply:
Sure, I suppose.